pages/common/b2-tools.md

```diff
@@ -0,0 +1,36 @@
+# b2-tools
+
+> Command-line tool that gives easy access to all of the capabilities of Backblaze B2 Cloud Storage.
```

Comment:
```suggestion
> Access all features of Backblaze B2 Cloud Storage easily.
```
We avoid putting "command-line" in the description and we prefer the imperative mood. See style guide.